drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -136,7 +136,7 @@ void ite_intc_irq_disable(unsigned int irq)
 	irq_unlock(key);
 }
 
-void ite_intc_irq_priority_set(unsigned int irq,
+void ite_intc_irq_polarity_set(unsigned int irq,
 		unsigned int prio, unsigned int flags)
```

Comment:
Should the `unsigned int prio` argument be renamed as well?

Reply:
The `prio` parameter is not unused.  Please remove it.

Reply:
prio is always passed as 0 and ignored in the function. Is there something coming that you will need this for?  I suggest remove or add a comment for this future usage.

Reply:
Done.